src/stores/orderStore.ts

```diff
@@ -208,10 +191,17 @@ class OrderStore extends BaseStore {
           Types.Expired
         );
       } else {
-        this.notificationStore.addNotification(
-          levels.error,
-          `${error.message}`
-        );
+        if (error.status === 500) {
+          this.rootStore.notificationStore.addNotification(
+            levels.error,
+            messages.defaultError
+          );
+        } else {
+          this.notificationStore.addNotification(
```

Comment:
What is the difference between this one and previous? (I think notificationStore should be take from one place)